CHANGELOG.md

```diff
@@ -1,5 +1,30 @@
 ## 10.4.0 (YYYY-MM-DD)
 
+All releases from 10.4.0 and forward are now found on `mavenCentral()` instead of `jcenter()`. 
+
+A minimal supported setup will therefore now look like this:
+
+```
+allprojects {
+    buildscript {
+        repositories {
+            mavenCentral()
```

Comment:
You would still need `jcenter()` until ReLinker is migrated too. Maybe only for dependencies, but at least one of the places.

Reply:
ReLinker has migrated: https://github.com/KeepSafe/ReLinker/issues/79#issuecomment-790133141

Reply:
Awesome. I just read the description of the PR in the feed, so didn't look up the status.